lib/grammars.coffee

```diff
@@ -199,11 +202,6 @@ module.exports =
         command: "bash"
         args: (context) -> ['-c', "xcrun clang++ -fcolor-diagnostics -Wc++11-extensions -Wall -include stdio.h -include iostream -framework Cocoa " + context.filepath + " -o /tmp/objc-cpp.out && /tmp/objc-cpp.out"]
 
-  ocaml:
-    "File Based":
```

Comment:
Whoops, looks like you don't have the latest code. Please remove this part of the change making sure you pull from rgbkrk/atom-script's master branch.